flink-core/src/main/java/org/apache/flink/api/common/io/BlockInfo.java

```diff
@@ -26,6 +26,11 @@
 import org.apache.flink.core.memory.DataOutputView;
 
 @Public
```

Comment:
`@Public` should go after Javadoc, IMHO. Also, the last line of the Javadoc has an extra `*`.